### DISCUSSION TITLE
onDayClick getting invoked twice when mode="range"?

### DISCUSSION THREAD

**Discussion by tchestnut85** [gpbl/react-day-picker#1475] [Support] [answered]
It looks like when the mode is set to `range` that onDayClick is getting invoked twice. 
Does anyone know if this is intentional (maybe since a range has a start and end)?

I modified one of the sandbox examples to reproduce this:

https://codesandbox.io/s/sandpack-project-forked-z0szxe?file=/App.tsx

Here's a recording too:

https://user-images.githubusercontent.com/67440557/175339913-6e3fab16-6ddd-457b-816c-045cf336d20f.mov

**Comment by gpbl** [comment, marked as answer]
Fixed in https://github.com/gpbl/react-day-picker/issues/1506
